.github/workflows/release.yml

```
@@ -83,7 +83,11 @@ jobs:
 
     - name: Install dependencies (Linux)
       if: startsWith(matrix.platform, 'ubuntu-')
-      run: sudo apt-get -y install ccache cmake make gcc g++ flex libfl-dev bison libpcap-dev libssl-dev python3 python3-dev python3-setuptools swig zlib1g-dev zip libmaxminddb-dev
+      run: |
+        sudo apt-get -y install ccache cmake make gcc g++ flex libfl-dev bison libpcap-dev libssl-dev python3 python3-dev python3-setuptools swig zlib1g-dev zip libmaxminddb-dev
+        sudo find /usr/lib \( -name libpcap.so\* -o -name libmaxminddb.so\* \) -delete
```

ZLIB and OpenSSL had available CMake hints to ask for static linking, but I couldn't find anything as simple to do the same for libpcap and libmaxminddb. So I found that if I just deleted all the shared libs on the Actions Runner before building, Zeek's `configure` would find the static `.a` libraries and happily link with them.

(FWIW I tried extending the trick to cover even ZLIB so I could avoid another hack described below. But when I deleted _that_ shared lib, a "compile a small test program" step early in Zeek's `configure` would fail.)

---

Nit: I'd add a comment to explain what's going on here.
```suggestion
        # Removing shared objects to force static linking.
        sudo find /usr/lib \( -name libpcap.so\* -o -name libmaxminddb.so\* \) -delete
```